content/6_Advanced/SCC.problems.json

```
@@ -177,7 +179,7 @@
       "isStarred": false,
       "tags": ["2SAT", "DSU", "Sliding Window", "Greedy"],
       "solutionMetadata": {
-        "kind": "internal"
+        "kind": "none"
```

```suggestion
        "kind": "autogen-label-from-site",
        "site": "CC"
```
you can autogen for codechef as well

---

actually never mind... i think only a couple of problems use it. i can still do the custom text for codechef though? wdyt

---

idk lol